test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -114,7 +114,9 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
 
         // TODO: Can we make this easier to re-invest fees, so that you don't need to know the exact collect amount?
         Plan memory planner = Planner.init();
-        planner.add(Actions.INCREASE_LIQUIDITY, abi.encode(tokenIdAlice, config, liquidityDelta, ZERO_BYTES));
+        planner.add(
+            Actions.INCREASE_LIQUIDITY, abi.encode(tokenIdAlice, config, liquidityDelta, 0 wei, 0 wei, ZERO_BYTES)
```

Comment:
how come this is 0 and not MAX_SLIPPAGE_INCREASE?

Reply:
we're ~perfectly autocompounding fees in this test so a strict slippage (no tokens paid by the caller) is an additional assurance